src/calculateRentalCost.js

```diff
@@ -28,7 +28,19 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const BASIC_PRICE_PER_DAY = 40;
+  const SMALL_DISCOUNT = 20;
+  const BIG_DISCOUNT = 50;
+
```

Comment:
```suggestion
const fullPrice = BASIC_PRICE_PER_DAY * days;
```
Move the repeating logic to the variable